src/wh_server_she.c

```diff
@@ -61,34 +57,65 @@ enum WOLFHSM_SHE_SB_STATE {
     WOLFHSM_SHE_SB_SUCCESS,
     WOLFHSM_SHE_SB_FAILURE,
 };
-uint8_t hsmShePrngState[WOLFHSM_SHE_KEY_SZ];
-uint8_t hsmSheSbState = WOLFHSM_SHE_SB_INIT;
-uint8_t hsmSheCmacKeyFound = 0;
-uint8_t hsmSheRamKeyPlain = 0;
-uint32_t hsmSheBlSize = 0;
-uint32_t hsmSheBlSizeReceived = 0;
-uint32_t hsmSheInitRng = 0;
+static uint8_t hsmShePrngState[WOLFHSM_SHE_KEY_SZ];
+static uint8_t hsmSheSbState = WOLFHSM_SHE_SB_INIT;
+static uint8_t hsmSheCmacKeyFound = 0;
+static uint8_t hsmSheRamKeyPlain = 0;
+static uint8_t hsmSheUidSet = 0;
+static uint32_t hsmSheBlSize = 0;
+static uint32_t hsmSheBlSizeReceived = 0;
+static uint32_t hsmSheRndInited = 0;
 /* cmac is global since the bootloader update can be called multiple times */
 Cmac sheCmac[1];
+Aes sheAes[1];
```

Comment:
Collect all required she server state into a single struct that is allocated/provided at the same time as the server.  Note that this state is per server instance and NOT global, but the server configs could use the same SHE instance to provide the same state, if desired.